webform_civicrm.module

```diff
@@ -27,16 +28,6 @@ define('WEBFORM_CIVICRM_DEFAULT_CONTACT_ID', 1);
 function webform_civicrm_menu() {
   /*
   $items = array();
-  $items['node/%webform_menu/civicrm'] = array(
-    'title' => 'CiviCRM',
-    'page callback' => 'drupal_get_form',
-    'page arguments' => array('wf_crm_configure_form', 1),
-    'access callback' => 'wf_crm_admin_access',
-    'access arguments' => array(1),
-    'file' => 'includes/wf_crm_admin_form.inc',
-    'weight' => 3,
-    'type' => MENU_LOCAL_TASK,
-  );
```

Comment:
It seems sort of odd to remove this item, when none of this code should be executed anyway? Not really problem, though.

Reply:
@mglaman This one is still unaddressed. Is this change even needed for anything? It seems super arbitrary, because this hook_menu() will never be run. If we're gonna remove something from hook_menu() why not the whole thing? Or just leave it be?